toctoc/core/src/test/scala/io.buildo.toctoc/core/authentication/TokenBasedRecoveryFlowSpec.scala

```diff
@@ -51,7 +51,7 @@ final class TokenBasedRecoveryFlowSpec extends munit.ScalaCheckSuite with CatsIO
           })
         } yield result).void.leftMap { e =>
           assertEquals(e, AuthenticationError.InvalidCredential)
-        }.value
+        }.swap.value
```

Comment:
What is `swap` for in those tests? Is it for improved debugging of failing tests?

Reply:
The general idea is that I want to return the left value, which is the content of `leftMap` above